flake.nix

```diff
@@ -118,10 +123,16 @@
         };
         modules = [
           {
+            enableLibraryProfiling = profiling;
             packages = {
               # This takes forever to build
               ghc.components.library.doHaddock = false;
             };
+            packages.inferno-core = {
```

Comment:
Could you format with `nixpkgs-fmt` for consistency with the rest of the Nix code please?